storage/src/cache/worker.rs

```diff
@@ -25,6 +25,8 @@ use crate::cache::{BlobCache, BlobIoRange};
 use crate::factory::ASYNC_RUNTIME;
 use crate::RAFS_MAX_CHUNK_SIZE;
 
+static PREFETCH_BEGIN_TIMING: Once = Once::new();
```

Comment:
This is a global variable, so it will only be executed once.
How about the case when prefetching multiple images?

Reply:
Thanks for your review, it's a good catch. Already fixed it.

In addition, added some comments telling the meaning of the newly added metrics and how to leverage them to estimate prefetch performance.